### ISSUE TITLE
lightweight-push stop working after alertR service update

### ISSUE THREAD

**Issue by pmendla** [sqall01/lightweight-push-pip#1]
When trying to send message:
10.Please make sure to use the newest version. If you do, please open an issue on https://github.com/sqall01/alertR .

**Comment by sqall01** [comment]
Yes, this is due to this issue: https://github.com/sqall01/lightweight-push/issues/1
